src/Operation/EstimatedDocumentCount.php

```diff
@@ -90,18 +115,53 @@ public function __construct($databaseName, $collectionName, array $options = [])
      */
     public function execute(Server $server)
     {
-        return $this->count->execute($server);
+        $command = $this->createCommand($server);
+        if ($command instanceof Aggregate) {
+            try {
+                $cursor = $command->execute($server);
+            } catch (CommandException $e) {
+                if ($e->getCode() == self::$errorCodeCollectionNotFound) {
+                    return 0;
+                }
+
+                throw $e;
+            }
```

Comment:
I'm not entirely happy with this part of the logic, but I couldn't come up with a nicer way to ignore nsNotFound errors.

Reply:
This is on par with what ListIndexes and DropCollection both do.